src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -84,6 +84,26 @@ public function testGuessRequired($constraint, $guess)
         $this->assertEquals($guess, $this->guesser->guessRequired(self::TEST_CLASS, self::TEST_PROPERTY));
     }
 
+    public function guessRequiredProviderWithGroup()
```

Comment:
I think we prefer putting data providers after the corresponding test cases

Reply:
done via 63959fdc43e527c477deda9f36c67c7b3e23379d